docs/reporters/notice-templates.md

```diff
@@ -6,9 +6,9 @@ customization of the generated open source notices with [Apache Freemarker](http
 
 ORT provides two templates that can be used as a base for creating your custom open source notices:
 
-* [default](../../reporter/src/main/resources/templates/notice/NOTICE_DEFAULT.ftl): Prints a summary of all licenses 
+* [default](../../reporter/src/main/resources/templates/plain-text/NOTICE_DEFAULT.ftl): Prints a summary of all licenses 
```

Comment:
Maybe move the changes in this file to the next commit, so all docs changes are in one commit? Together with my comment about the shell scripts there would be three commits then: Code changes, completion shell regeneration, doc changes.